src/utils/junit/JUnitCommandHandler.ts

```diff
@@ -76,7 +76,8 @@ export class JUnitCommandHandler {
 					for (const attachment of result.attachments) {
 						if (attachment.buffer) {
 							const { url } = await this.api.file.uploadFile(
-								new File([attachment.buffer], attachment.filename)
```

Comment:
File is a fairly new feature which is went out of experimantal since Node version 20. Using covers older versions.